crates/ruff_python_formatter/resources/test/fixtures/ruff/statement/match.py

```diff
@@ -168,6 +168,18 @@ def foo():
         pass
 
 
+match pattern_comments:
+    case (
+        pattern
+        # 1
+		as
+        # 2
+		name
+        # 3
+    ):
+        pass
```

Comment:
Could we add a test case with just end of line comments?
```python
match subject:
    case (
        pattern # 1
        as # 2
        name # 3
    ):
        pass
```

Black collapses them but I don't think we should.